apps/visualization/utils.py

```diff
@@ -556,3 +559,24 @@ async def get_global_combined_indicators(filters):
     if filters:
         qs = filter_apply(filters, qs)
     return await process_combined_indicators(qs, type=GLOBAL_LEVEL)
+
+
+@sync_to_async
+def get_export_meta_data(iso3, indicator_id):
+    from .types import ExportMetaType
+    filters_map = {
+        'iso3': iso3,
+        'indicator_id': indicator_id,
+    }
+    contextual_filters_map = {
+        'iso3': iso3,
+        'context_indicator_id': indicator_id,
+    }
+    filters = {k: v for k, v in filters_map.items() if v is not None}
+    contextual_filters = {k: v for k, v in contextual_filters_map.items() if v is not None}
```

Comment:
Let's create a function for this. `def remove_empty_value_from_dict`

Reply:
@thenav56  I added this